examples/helm-template/README.md

```diff
@@ -15,8 +15,131 @@ kpt fn run helm-template/local-configs --mount type=bind,src=$(pwd)/helm-templat
 
 ## Expected result
 
+Checking the contents of the `local-configs` directory should reveal the following new yaml files:
+
+```sh
+./local-configs:
```

Comment:
Nit: Use `kpt cfg tree`? It looks better if we can list these one file per line.

Reply:
+1

Reply:
`kpt cfg tree` on the example output gives me `error: /tmp/helm-template/helloworld-chart/templates/deployment.yaml: yaml: line 5: did not find expected node content`

Reply:
Done.